components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -1721,94 +1723,22 @@ private void updatePermissionAndRoleConfiguration(int applicationID,
     }
 
     @Override
-    public ServiceProvider getApplication(String applicationName, String tenantDomain)
-            throws IdentityApplicationManagementException {
+    public ServiceProvider getApplication(String applicationName,
```

Comment:
Refactored this method to remove code duplication. We already have a method that loads an SP by internal id.